src/Controllers/MovieController.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.EntityFrameworkCore;
+using TrainingNet.Models.DataBase;
+using TrainingNet.Repositories;
+using TrainingNet.Repositories.Database;
+using TrainingNet.Repositories.Interfaces;
+
+namespace TrainingNet.Controllers
+{
+    [Route("[controller]")]
+    public class MovieController : Controller
+    {
+        private readonly IHtmlLocalizer<HomeController> _localizer;
+        private readonly IUnitOfWork _unitOfWork;
+        public MovieController(IHtmlLocalizer<HomeController> localizer, IUnitOfWork unitOfWork){
+            this._localizer = localizer;
+            this._unitOfWork = unitOfWork;
+        }
+        [HttpGet("")]
+        [HttpGet("Index")]
```

Comment:
if this method and view is to create a movie... it doesn't make sense to call it "Index" or "AddMovie". you can use "Add" or "Create" because method it's already on movie controller